dashboard/app/controllers/api/v1/user_school_infos_controller.rb

```diff
@@ -0,0 +1,69 @@
+class Api::V1::UserSchoolInfosController < ApplicationController
+  protect_from_forgery with: :null_session
```

Comment:
What does this do?

Reply:
[See](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/controllers/application_controller.rb#L16)

In the application controller, the setting is "protect_from_forgery with: :exception", which raises an exception if there is a CSFR type request.  

protect_from_forgery with: :null_session does not raise an exception but will nullify the session for the duration of the request.  At a certain point, I needed to set "protect_from_forgery to null_session" to get my test to pass but no longer require it.